src/utils/cmds/flushstate.c

```diff
@@ -0,0 +1,106 @@
+#include "collectd.h"
+
+#include "daemon/utils_cache.h"
+#include "plugin.h"
+#include "utils/cmds/flushstate.h"
+#include "utils/common/common.h"
+
+cmd_status_t cmd_parse_flushstate(size_t argc, char **argv,
+                                  cmd_flushstate_t *ret_flushstate,
+                                  const cmd_options_t *opts,
+                                  cmd_error_handler_t *err) {
+  char *identifier_copy;
+
+  int status;
+
+  if ((ret_flushstate == NULL) || (opts == NULL)) {
+    errno = EINVAL;
+    cmd_error(CMD_ERROR, err, "Invalid arguments to cmd_parse_flushstate.");
+    return CMD_ERROR;
+  }
+
+  if (argc != 1) {
+    if (argc == 0)
+      cmd_error(CMD_PARSE_ERROR, err, "Missing identifier.");
+    else
+      cmd_error(CMD_PARSE_ERROR, err, "Garbage after identifier: `%s'.",
+                argv[1]);
+    return CMD_PARSE_ERROR;
+  }
+
+  identifier_copy = sstrdup(argv[0]);
```

Comment:
You can define variable at first use
`char *identifier_copy = sstrdup(argv[0]);`